features/images.feature

```diff
@@ -0,0 +1,40 @@
+@images
+Feature: OpenStack Image management
+  As an administrator
+  I want to make sure that OpenStack Image management component
+  is working correctly
+
+  @create_kvm
+  Scenario: Creation of KVM image
+    Given KVM source image exists
+    And glance image does not exist
+    When I create new glance image based on jeos
+    Then this image has non-empty ID
+    And its ID can be used to show the image info
+    And the status of the image is active
+    When I delete the image
```

Comment:
Actually the DRY violation is a bit worse than I described, because these 3 steps are also duplicated in every scenario:

```gherkin
Then this image has non-empty ID
  And its ID can be used to show the image info
  And the status of the image is active
```

So it should be possible to collapse each scenario from 8 steps down to 2 or 3 by using appropriate nested steps.

One more point - these are mostly not my original ideas.  Lots of people advocate DRY in Cucumber features, e.g.

* http://eggsonbread.com/2010/09/06/my-cucumber-best-practices-and-tips/
* http://makandracards.com/makandra/18905-how-to-not-repeat-yourself-in-cucumber-scenarios
* http://rails-bestpractices.com/tag/cucumber/
* http://coryschires.com/ten-tips-for-writing-better-cucumber-steps/